pre-commit/package_app_dependencies

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+APP_DIR=$(pwd)
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+if ! docker info &> /dev/null; then
+  echo 'Please ensure Docker is installed and running on your machine'
+  exit 1
+fi
+
+docker run --rm -v "$APP_DIR":/src -v "$SCRIPT_DIR":/pre-commit/ -w "$SCRIPT_DIR" \
+  quay.io/pypa/manylinux2014_x86_64 /bin/bash -c \
+   "/opt/python/cp39-cp39/bin/python /pre-commit/package_app_dependencies.py /src"
```

Comment:
how long does this take? I think you may have said this only takes a few seconds during standup but just wanted to confirm that's the case since it's going to run every time someone tries to make a commit

Reply:
Running on `awslambda` takes 9 seconds - the hook will only run when `requirements.txt` is modified